packages/terra-layout/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "terra-layout",
+  "main": "lib/Layout.js",
+  "version": "0.0.0",
+  "description": "This component renders navigation with react router.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-framework.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "Clinical",
+    "terra-clinical-layout",
+    "Layout",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-clinical/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-clinical#readme",
+  "peerDependencies": {
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2",
+    "terra-base": "^2.5.0"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "prop-types": "^15.5.8",
+    "terra-base": "^2.5.0",
+    "terra-button": "^1.9.0",
+    "terra-content-container": "^1.9.0",
+    "terra-icon": "^1.10.0",
+    "terra-overlay": "^1.9.0",
+    "terra-responsive-element": "^1.9.0"
+  },
+  "scripts": {
+    "compile": "npm run compile:clean && npm run compile:build",
+    "compile:clean": "rimraf lib",
+    "compile:build": "babel src --out-dir lib --copy-files",
+    "lint": "npm run lint:js && npm run lint:scss",
+    "lint:js": "eslint --ext .js,.jsx . --ignore-path ../../.eslintignore",
+    "lint:scss": "stylelint src/**/*.scss",
+    "props-table": "props-table ./src/ActionHeader.jsx --out-dir ./docs/props-table",
```

Comment:
Update ActionHeader.jsx to the `.jsx` files you want to generate props documentation for.

Reply:
Updated 👍